src/Http/Client/Adapter/Curl.php

```diff
@@ -44,7 +44,7 @@ public function send(Request $request, array $options)
             curl_close($ch);
 
             $status = 500;
-            if ($error === 28) {
+            if ($errorCode === 28) {
                 $status = 504;
```

Comment:
Too much magic numbers here:
- `CURLE_OPERATION_TIMEDOUT (28)`
- `500 Internal Server Error`
- `504 Gateway Timeout`

Reply:
I replaced `28` with relevant constant. `Client\Response` doesn't have any constants for `500` and `504`.

Reply:
Simple comment would help.

To understand these lines, both 28 and 504 need constants or explanations.

500 is more common code, description is optional/unnecessary, especially in context of Curl/HTTP Client.

Reply:
Anyone who you care to look into this code would know where to get details about `504` HTTP status code too :slightly_smiling_face:.